Frontend/src/components/Admin/Users/AdminListUsers.tsx

```diff
@@ -75,6 +77,7 @@ export default () => {
                 <Button onClick={() => setDeleteDialogOpen(false)}>Cancel</Button>
             </DialogActions>
         </Dialog>
+        <TextField label="Search user" type="text" value={search} onChange={(ev) => { setSearch(ev.target.value); setPage(0); }} />
```

Comment:
I would move these to one combined function to keep the jsx clean

Reply:
Alright, refactored a few other similar things into functions as well then.